docs/mv3-architecture.md

```diff
@@ -27,26 +27,26 @@ This is the page being tested by the extension. The extension injects content sc
 
 #### Popup
 
-The page/ javascript thread that is opened when the extension is activated.
+The page/javascript thread that is opened when the extension is activated.
 
 ![Screenshot of popup](./screenshots/popup.png)
 
-#### Background/ Service Worker
+#### Background/Service Worker
 
-A javascript thread that runs for the extension outside of the scope of any specific tab. With manifest v2, this thread was called a [background page or background script](https://developer.chrome.com/docs/extensions/mv2/background_pages/). It was persistent and could be depended on to be the always available 'source of truth' for the whole system. With manifest v3, background pages were replaced by [service workers](https://developer.chrome.com/docs/workbox/service-worker-overview/), the main difference being that service workers have finite lifetimes and shut down once the extension goes idle after a certain timeout.
+A javascript thread that runs for the extension outside of the scope of any specific tab. With manifest v2, this thread was called a [background page or background script](https://developer.chrome.com/docs/extensions/mv2/background_pages/). It was persistent, always available, and considered the singular 'source of truth' for the whole system. That is to say, the background page has a 1:many relationship with the tab contexts it is managing. With manifest v3, background pages were replaced by [service workers](https://developer.chrome.com/docs/workbox/service-worker-overview/), the main difference being that service workers have finite lifetimes and shut down once the extension goes idle after a certain timeout.
```

Comment:
> That is to say, the background page has a 1:many relationship with the tab contexts it is managing

This seems to imply that the service worker differs when it doesn't, can we move this sentence to the end of the paragraph and explicitly include the service worker- `... the background page and service workers have a...`? And maybe we should include a note about there only being one service worker at a time?

Reply:
Hm, I thought the "main difference" line might imply that is unchanged for service workers but maybe it makes sense to be more specific. I'll add the note about the service workers as well.